.travis.yml

```diff
@@ -7,7 +7,13 @@ addons:
     packages:
       - libgnutls-dev
 
-# For a list of available versions, run
+env:
+  global:
+    - CIBW_ENVIRONMENT="PYCURL_SSL_LIBRARY=nss"
+    - CIBW_BEFORE_BUILD="yum install -y libcurl-devel openssl-devel"
```

Comment:
Why do we install `openssl-devel` but tell pycurl to use `nss`? 

This could go in `CIBW_BEFORE_TEST` instead of `CIBW_BEFORE_BUILD`. Our build itself doesn't need any extra dependencies beyond what CIBW provides.

Reply:
Yes, `openssl-devel` is not required here. In fact whole Global env section is not required now, as we are testing the built wheel using `"tox --installpkg"`.